torch/_subclasses/meta_utils.py

```diff
@@ -370,43 +407,58 @@ class MetaStorageDesc:
     # serializable in JSON, you want to do something special here anyway
     data: Optional[torch.UntypedStorage]
 
+    def as_json(self, describer_id):
+        return {
+            "id": self.id,
+            "describer_id": describer_id,
+            "size": self.size if isinstance(self.size, int) else repr(self.size),
+        }
+
 
 @dataclass(frozen=True)
 class MetaTensorDesc:
     id: MetaTensorId
-    is_inference: bool
-    is_leaf: bool
-    requires_grad: bool
     ndim: int
     dtype: torch.dtype
-    is_sparse: bool
-    is_mkldnn: bool
-    is_functorch_wrapped: bool
-    is_batchedtensor: bool
-    is_legacy_batchedtensor: bool
-    is_gradtrackingtensor: bool
-    is_view: bool
-    is_nested: bool
-    is_traceable_wrapper_subclass: bool
-    is_functional: bool
-    is_conj: bool
-    is_neg: bool
     device: torch.device
-    layout: torch.layout
+
     # NB: Sometimes, size, stride and storage_offset contain SymInt, in which
```

Comment:
(Unrelated) I actually ran into this trying to serialize/unserialize fake tensors that are used as metadata for caching AOTAutograd... luckily in my case those fields don't actually matter but definitely took me a while to debug what was happening...